benchmark-framework/src/main/java/io/openmessaging/benchmark/RateController.java

```diff
@@ -68,7 +69,17 @@ class RateController {
             return nextRate(periodNanos, published, expected, publishBacklog, "Publish");
         }
 
-        rampUp();
+        if (targetP99EndToEndLatency != 0 && targetP99PublishLatency != 0) {
```

Comment:
Do we have to enable both targetP99EndToEndLatency and targetP99PublishLatency to enable this `latency-based max-rate feature`?

Reply:
I would like to talk more about this in the slack thread.